### PR TITLE
Add solution to problem 22 of Project Euler

### PR DESCRIPTION
**Pull request by anubhav8421**

### CODE REVIEW

**Review by vzvu3k6k — APPROVED** [review]
Nice work :+1: 

I've added some very minor comments as there seems some room to simplify the code.

File: Project Euler/Problem 22/problem22_sol1.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+file_contents = File.open('p022_names.txt', 'r').read
+
+# replacing the occuerance of \" to '' and spliting the result by ','
+# to get an array of sorted words
+words = file_contents.tr('\"', '').split(',').sort
+
+# this method calculates the worth of a word based on the ASCII
+# values of the characters
+def word_worth(word)
+  word.split('').inject(0) { |result, char| result + char.ord - 'A'.ord + 1 }
```
[the remark this answers is not in the document]
Comment (anubhav8421) [reply Project Euler/Problem 22/problem22_sol1.rb]: The changes wrt this have been done in the latest commit added to the PR

[the remark this answers is not in the document]
Comment (anubhav8421) [reply Project Euler/Problem 22/problem22_sol1.rb]: The changes wrt this have been done in the latest commit added to the PR

**Comment by anubhav8421** [comment]
@vzvu3k6k this PR is ready to be reviewed again

**Review by vzvu3k6k — APPROVED** [review]
LGTM :+1:

**Comment by anubhav8421** [comment]
@AnupKumarPanwar @vzvu3k6k When can this be merged?

**Comment by vzvu3k6k** [comment]
@anubhav8421 
Oops... I'm not a committer of this repository. I just happened to be interested in this project and your work. So you don't have to follow my comments to be merged.

I should have made it clear in advance. Sorry for confusing you.

**Comment by anubhav8421** [comment]
No worries. I will try to contact the maintainers

**Pull request merged**